### ISSUE TITLE
Detect location based on photo

### ISSUE THREAD

**Issue by peterdesmet** [inbo/vespa-watch#237]
@KarelSchoonvaere asked if the location of the observation can be detected from the geotag of the photo. That would be neat indeed.

**Comment by timadriaens** [comment]
indeed, it is standard in iNaturalist and it helps us validating or flagging dubious records in case someone uploads a false observation with a picture they found online somewhere.

**Comment by KarelSchoonvaere** [comment]
From my experience most pictures take by observers do not have geo-tags (unless my outlook removes the tags). But good point Tim

**Comment by timadriaens** [comment]
Unless they use their smartphone...

Op do 28 mrt. 2019 17:56 schreef KarelSchoonvaere <notifications@github.com

**Comment by niconoe** [comment]
Can also work for time!

**Comment by peterdesmet** [comment]
Won't implement this soon, moving this to `parked`
